kola/tests/kubeadm/kubeadm.go

```diff
@@ -76,12 +76,13 @@ systemd:
 
 func init() {
 	for _, CNI := range CNIs {
+		cni := CNI
```

Comment:
I see now that this is variable capturing in the `Run` function that bit us here. Nasty. Would a comment about the fact be helpful here?

Reply:
Yes for sure. Actually, this section will be replaced by a new one from this PR (#196) - and a comment is already there to explain while we need such a thing (see https://github.com/kinvolk/mantle/pull/196/files#diff-833c725698d5fe10161a1c491a560567f1473d7541658c722022bc45db2d4c3fR88-R95)

Reply:
Ah, cool. Thanks.